blog/2019/01/fluxdiffeq.md

```diff
@@ -331,7 +323,7 @@ we can stick it right in there:
 m = Chain(
   Dense(28^2, 32, relu),
   # this would require an ODE of 32 parameters
-  p -> concrete_solve(prob,Tsit5(),u0,p,saveat=0.1)[1,:],
+  p -> solve(prob,Tsit5(),saveat=0.1)[1,:],
```

Comment:
should be `p -> solve(prob,Tsit5(),p=p,saveat=0.1)[1,:],`?

Reply:
yes